locales/en-US/app.ftl

```diff
@@ -132,7 +134,8 @@ disableExperimentTransition = Disabling...
 enableExperiment = Enable {$title}
 enableExperimentTransition = Enabling...
 experimentManuallyDisabled = {$title} disabled in Add-ons Manager
-experimentMeasurementIntro = In addition to the <a>data</a> collected by all Test Pilot experiments, here are the key things you should know about what is happening when you use { $experimentTitle }:
+experimentMeasurementIntro2 = In addition to the {$data} collected by all Test Pilot experiments, here are the key things you should know about what is happening when you use {$experimentTitle}:
+experimentMeasurementIntroDataLink = data
```

Comment:
Please add a localization comment explaining how this is used. Remember that localizers see one string at a time, not an entire file.

Same comment is valid for other places where this happens.